flax/configurations.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Global configuration options for Flax.
+r"""Global configuration options for Flax.
```

Comment:
Not sure why I needed to add this but Python was refusing to run this line due to a strange character.